google_compute_engine_oslogin/pam_module/pam_oslogin_login.cc

```diff
@@ -49,20 +49,20 @@ PAM_EXTERN int pam_sm_acct_mgmt(pam_handle_t *pamh, int flags, int argc,
 
   std::stringstream url;
   url << kMetadataServerUrl << "users?username=" << UrlEncode(str_user_name);
-  string response = HttpGet(url.str());
-  if (response == "") {
+  string response;
+  if (!HttpGet(url.str(), &response) || response.empty()) {
+    // If we are not dealing with an oslogin, the default behavior is to permit.
```

Comment:
If we return DENIED, then in the event of a persistent backend failure, we block out ALL users, even local ones. No one will be able to log into the VM.